packages/react-next/src/components/Checkbox/Checkbox.scss

```diff
@@ -269,18 +271,18 @@ $checkboxTransitionTiming: cubic-bezier(0.4, 0, 0.23, 1);
     color: WindowText;
   }
 
-  &.disabled {
+  .disabled & {
     color: var($semanticColorsDisabledText);
 
     @include high-contrast {
       color: GrayText;
     }
   }
 
-  &.reversed {
+  .reversed & {
     margin-right: 4px;
   }
-  &:not(.reversed) {
+  :not(.reversed) & {
```

Comment:
You might want to double check that this works as expected. I found I had a lot of difficulties getting `:not(.name) &` to work appropriately.

Reply:
could you elaborate on what exactly wasn't working for you?

Reply:
Yeah unfortunately it doesn't seem to behave as expected :( https://codepen.io/slimemold/pen/YzyRvxr

Reply:
I missed your response! It seems like using `:not(.classname) &` doesn't actually check the parent. In the scss "preview" in vs code it seems to parse away the `classname` so it's not applying the not to the parent properly 

![image](https://user-images.githubusercontent.com/20801176/82265978-3aee0480-991d-11ea-92b1-486c07217324.png)

You can see this in the codepen that I linked above, the `:not(classname) &` is applied to the classname it's contained in, rather than the parent.

![image](https://user-images.githubusercontent.com/20801176/82266005-56590f80-991d-11ea-9f84-e81ac39b96dd.png)

Reply:
AH! yeah this won't work of course. becasue `.notchecked` here definitely satisfies having ancestor `:not(.checked)`

Reply:
will close the PR. @joschect  - could you fix your PR accordingly too?